gluten-arrow/src/main/scala/org/apache/gluten/runtime/Runtime.scala

```diff
@@ -47,7 +47,7 @@ object Runtime {
       nmm.getHandle(),
       ConfigUtil.serialize(
         GlutenConfig
-          .getNativeSessionConf(backendName, GlutenConfigUtil.parseConfig(SQLConf.get.getAllConfs)))
+          .getNativeSessionConf(backendName, GlutenConfigUtil.parseConfig(getConf)))
```

Comment:
@taiyang-li Is the way clickhouse backend passes SQLConf different from velox? Can you help take a look? Thank you.